zproject/prod_settings_template.py

```diff
@@ -211,6 +212,139 @@
+#     BQADggEPADCCAQoCggEBAM9DeYmZ9ur9BTH3DHky+F+xElcIhRmXAVo9XS4GILJ/
+#     emd5PM47aNWKi75bNailyQ3F7IZn3k8paoQ206DwrtgCGaKKcgXF5JbsQqATBvBU
+#     1CXQtakvwAQkxxz5kKcnaru0eJR/BCp0fsTo7Z4DiptheohBJMkcvi9gx1ZDDeC6
+#     GWh1mAjYY6UyW7cXtC3vywASrLMWEFUf4n/mrl9iyPeEPyn9RZaOhMObySE/1EF3
+#     IKYzKWzRFcpMMiNSRCCbtBO+ojL34+ixEr3m5YZlboC6GsDB1goXBG5ChuX3DQ4n
+#     Wt6WP7TBklz+x3Ki8hJKy59qmPCh2xQtMnSGUefgh9MCAwEAAaNTMFEwHQYDVR0O
+#     BBYEFFkn0vZOC5Uu08DmLEDjbgCbxY0KMB8GA1UdIwQYMBaAFFkn0vZOC5Uu08Dm
+#     LEDjbgCbxY0KMA8GA1UdEwEXGGFGFGFEEREgDQYJKoZIhvcNAQELBQADggEBAF8g
+#     wHMsSUhbgUBGQ5ePCO3g8d0qoYzhfFOCKJbajZtyhOfL3xCoRPO7fbeWD2oHc1Ey
+#     hE9p87X8evqzTjHtLzRt76I2b/2Vqj9CSwVskSAvYuYKqztK2SYd8mbAhTZ2ORSj
+#     J9hFfgfHFpF1VSQfp9p5OZVrLQHh68zFGROPIjpoPuVdz4WTqYtayH55cIwBo+Sk
+#     tPIX/MGYQAH8ex9uEkbuiiXxs+n2iLcGNCKet9i5tal2/x5ERD9Il+zJWjVV/eDd
+#     7H9/CJrCmM4qsSPGQrEAKJ9zt92KZQ/zA2haCrIiuR4aaWfqaXKfrGmC4ZA3mj0l
+#     YJx5jA27BC6tmaGOIi0=
+#     """.replace(' ', '').replace('\n', '').strip()
+#
+# SOCIAL_AUTH_SAML_SP_PRIVATE_KEY = \
+#     """
+#     MIIEvgIBADANBgkqhkiG9w0BAQEFAASCBKgwggSkAgEAAoIBAQDPQ3mJmfbnfQUx
+#     9wx5MvhfsRJXCIUZlwFaPV0uBiCyf3pneTzOO2jViou+WzWopckNxeyGZ95PKWqE
+#     NtOg8K7YAhmiinIFxeSW7EKgEwbwVNQl0LWpL8AEJMcc+ZCnJ2q7tHiUfwQqdH7E
+#     6O2eA4qbYXqIQSTJHL4vYMdWQw3guhlodZgI2GOlMlu3F7Qt78sAEqyzFhBVH+J/
+#     5q5fYsj3hD8p/UWWjoTDm8khP9RBdyCmMyls0RXKTDIjUkQgm7QTvqIy9+PosRK9
+#     5uWGZW6AuhrAwdYKFwRuQobl9w0OJ1relj+0wZJc/sdyovISSsufapjwodsULTJ0
+#     hlHn4IfTAgMBAAECggEAYgUVK4BHBn9Xv6OIoKD8WbUSaVq8J7gEibrPoH6BZClp
+#     qBnsKdRq5eKZ6DKWk+WjygG6tG3W8zUxDMopF8n7xmhn2zkCZ7yJON8PO0kEVRCp
+#     LCEiaJsOgmcqS4ezY3n+vjrBc+kF/VFRU654HHoFiSp2mo3EaeBQCXQJkcAwhgBi
+#     RP38z3c9A+I3APZ19EJ+ByNvLpo97VvUZjPMjIxF9CiniCLH9k/Humz6EiC6B5xS
+#     l3xKNakBW0wSN9ontAnWpdRI9KkSy48lxcqAkQyrueOVRTOAsZiJ1WzgAplRSdUD
+#     4W6NOmNKayeRL9SMh/h0xxBrh/iFWRKva9kbh1JgQQKBgQD82osiEUitv2++sG3t
+#     4lqGcGEWqrECG/bhG4d7jAKNeJzl7UIC6+P9EFgdh9tZ1W1rSzYWe2mO2kj1GeQZ
+#     fn8ZDQKJxQX5M796D2L00obMdRhGhkRDunsWS7HGDDc42GMVltrRZQ7b0RwOAVqc
+#     XZqWQrkg6I+A1VWDdp5IfExakQKBgQDR17SgPpgU9PsaJnNm4+M4W0hF0IwztXyU
+#     JkYqYaZrO6YEZ2ozA4rQCWPpdHcYHNueKfOxi6KkMNbWppNa7yUoK5CtGHlH2qQt
+#     BHrNVqQQyVAQtzzy4/FGGGfozMFaTe6UOO1hQNfK1Bd7+qSXnRcvCgVIc+BaqWzI
+#     bVugryTGIwKBgGMpukMEEELDvylDu/j9Oo9a51iiXIT9PpcN2tZjFGUzFiU+JRNM
+#     O791d3ktXKMghqb7ehzbDYzOoqotGz28VFAYwBQTxsHyv3c5LY/iOKdajMirw33k
+#     Vn1o2FtRM1bh1goTfYF9ZpRir+P2N/j25LdlN4Fmc1ENWVoTXQQEa1VhAoGBAMqr
+#     nF/ctMQg7FkQZAsu9OkceMw9JPxYP+9SnBgY4kPVg33jZ9XHiQ1u1FnJS7E1RgQX
+#     nq+PHtGX/hepIM8GBIwsr3oPFiwJBuIX3oduM92mobbTU4GAGCrMLafSArk2ZcQw
+#     y0xP9VuHxvrOVKO+bILZGcslZ74IvvFBderW3hyjAoGBAMEkcZLfe2+FmAe2fju0
+#     F1qCAWqnqpVAgWsuDycS6k5TYHj/6vm8XVx+Xa8r/IfTuyXlAq/CPb4KPPY4ZAUy
+#     iAHVqzslbBuBOHaukySepy2Z4GBaMoD742H0QBdaNQE8pBjZhyir+paTbkehd5kT
+#     8LJ2ctg7ipCEcGe5b+gEllTe
+#     """.replace(' ', '').replace('\n', '').strip()
```

Comment:
We generally don't want secrets to live in `/etc/zulip/settings.py` from a security model perspective.  See my next comment for a proposal.